src/main/java/com/zendesk/maxwell/replication/BinlogConnectorEventListener.java

```diff
@@ -41,6 +43,11 @@ public void stop() {
 	public void onEvent(Event event) {
 		long eventSeenAt = 0L;
 
+		if (event.getHeader().getEventType() == EventType.QUERY) {
```

Comment:
We already have a local `trackMetrics` below, for XID events. I would assume execution time is per transaction, not per update - can we track this only for XIDs, or is that not how it works?